webapp/platform/client/package.json

```diff
@@ -35,5 +35,8 @@
     "test": "jest",
     "test-ci": "jest --ci --forceExit --detectOpenHandles --maxWorkers=100%",
     "clean": "rm -rf lib node_modules tsconfig.tsbuildinfo"
+  },
+  "devDependencies": {
```

Comment:
we hd a little bit of reasoning why we had this removed from here which I am not able to recollect